xml_analyzer/xmlreader.py

```diff
@@ -1,38 +1,37 @@
 import collections
 import xml.etree.ElementTree as Et
 from .data_container import DataContainer
+from .definitions import AutocalcMethods
 
 
 class XMLReader:
-    autocalc_methods = ['autosum']
 
     def __init__(self, project_file):
         self._project_file = project_file
         self.tree = Et.parse(self._project_file)
         self.root = self.tree.getroot()
-        self.pv_map = collections.namedtuple('property_value_mapping', ['property', 'value'])
 
     @staticmethod
-    def parse_constraint_spec(constraint_spec: str) -> list:
+    def _parse_constraint_spec(constraint_spec: str) -> list:
         autocalc_sysml_type_name = list()
-        for method in XMLReader.autocalc_methods:
-            if method in constraint_spec:
-                autocalc_sysml_type_name.append(method)
-                for occurrence in constraint_spec.split(method)[1:]:
+        for method in AutocalcMethods:
+            if method.name in constraint_spec:
+                autocalc_sysml_type_name.append(method.name)
+                for occurrence in constraint_spec.split(method.name)[1:]:
                     search_param = occurrence[occurrence.find('(') + 1:occurrence.find(')')]
                     if search_param != '':
```

Comment:
Can be simplified into 
`if search_param:
`

Reply:
Fixed with https://github.com/Squiddrone/VPParametricsSolver/pull/2/commits/125a9de3f5317b3cc4f907790a9fabccc9134c0e